eng/build-job.yml

```diff
@@ -22,10 +22,7 @@ jobs:
     name: ${{ format('build_{0}_{1}_{2}', parameters.osIdentifier, parameters.archType, parameters.buildConfig) }}
     displayName: ${{ format('Build {0} {1} {2}', parameters.osIdentifier, parameters.archType, parameters.buildConfig) }}
 
-    # FreeBSD builds are disabled in the public project because we
-    # don't have any FreeBSD agents in the public pool.
-    ${{ if and(eq(parameters.osGroup, 'FreeBSD'), eq(variables['System.TeamProject'], 'public')) }}:
-      condition: false
+    condition: ${{ parameters.condition }}
```

Comment:
Is this needed? `condition' isn't listed as parameter at the top and nothing seems to explicitly set this when using `build-job`?

Reply:
You're right. I think this was me simply removing the FreeBSD settings. I am going to remove this entirely.